index.html

```diff
@@ -611,16 +610,16 @@ <h2>Terminology</h2>
 A set of one or more [=claims=] made by an [=issuer=]. The [=claims=]
 in a credential can be about different [=subjects=]. The definition of
 credential used in this specification differs from,
-<a href="https://csrc.nist.gov/glossary/term/credential">NIST's definitions of
+<a href="https://csrc.nist.gov/glossary/term/credential">NIST's definition of
```

Comment:
```suggestion
<a href="https://csrc.nist.gov/glossary/term/credential">NIST's definitions of
```

NIST actually has multiple different definitions for the word "credential" :)

Reply:
Note that preceding line, `credential used in this specification differs from,` should not end with a comma, i.e., "differs from, NIST's definitions" should be "differs from NIST's definitions".